src/main/java/net/dv8tion/jda/internal/entities/PrivateChannelImpl.java

```diff
@@ -82,6 +82,12 @@ public long getLatestMessageIdLong()
         return latestMessageId;
     }
 
+    @Override
+    public boolean canTalk()
+    {
+        return true;
```

Comment:
@MinnDevelopment correct me if I'm wrong, but this should probably be `!user.isBot()`, right?

Reply:
I guess we could, as long as the docs outline that we can't check for blocked users